code/game/cas_manager/datums/cas_fire_envelope.dm

```diff
@@ -259,7 +259,26 @@
 		mission_error = "Target is off bounds or obstructed."
 		return
 	change_current_loc(target_turf)
-	playsound(target_turf, soundeffect, 70, TRUE, 50)
+	playsound(source = target_turf, soundin = soundeffect, vol = 70, vary = TRUE, sound_range = 50, falloff = 8)
+
+	var/relative_dir
+	for(var/mob/mob in range(15, target_turf))
+		if(get_turf(mob) == target_turf)
+			relative_dir = 0
+		else
+			relative_dir = Get_Compass_Dir(mob, target_turf)
+
+		var/ds_identifier = "LARGE BIRD"
+		var/fm_identifier = "SPIT FIRE"
+		if (mob.mob_flags & KNOWS_TECHNOLOGY)
+			ds_identifier = "DROPSHIP"
+			fm_identifier = "EXECUTE FIREMISSION"
+
+		mob.show_message( \
+			SPAN_HIGHDANGER("YOU HEAR [ds_identifier] ROAR AS IT PREPARES TO [fm_identifier] [SPAN_UNDERLINE(relative_dir ? uppertext(("TO YOUR " + dir2text(relative_dir))) : uppertext("right above you"))]!"), SHOW_MESSAGE_VISIBLE, \
```

Comment:
```suggestion
			SPAN_HIGHDANGER("YOU HEAR THE [ds_identifier] ROAR AS IT PREPARES TO [fm_identifier] [SPAN_UNDERLINE(relative_dir ? uppertext(("TO YOUR " + dir2text(relative_dir))) : uppertext("right above you"))]!"), SHOW_MESSAGE_VISIBLE, \
```
`"YOU HEAR LARGE BIRD ROAR..."`/`"YOU HEAR DROPSHIP ROAR..."`
vs
`"YOU HEAR THE LARGE BIRD ROAR..."`./`"YOU HEAR THE DROPSHIP ROAR..."`
<hr>
Alternatively "A" would work instead of "The".